homeassistant/components/intellifire/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the IntelliFire integration."""
+from __future__ import annotations
+
+import logging
+
+DOMAIN = "intellifire"
+
+_LOGGER = logging.getLogger(__package__)
```

Comment:
`_` indicates a protected thing, which isn't the case here.

```suggestion
LOGGER = logging.getLogger(__package__)
```